dagger-core/src/test/java/io/odpf/dagger/core/sink/SinkOrchestratorTest.java

```diff
@@ -124,8 +127,8 @@ public void shouldReturnOutputKafkaMetrics() {
         when(configuration.getString(eq("SINK_KAFKA_BROKERS"), anyString())).thenReturn("output_broker:2667");
         when(configuration.getString(eq("SINK_KAFKA_STREAM"), anyString())).thenReturn("test_output_stream");
         when(configuration.getString(eq("SINK_KAFKA_TOPIC"), anyString())).thenReturn("test_topic");
+        when(configuration.getString(eq("SINK_KAFKA_DATA_TYPE"), anyString())).thenReturn("PROTO");
 
         sinkOrchestrator.getSink(configuration, new String[]{}, stencilClientOrchestrator);
-        assertEquals(expectedMetrics, sinkOrchestrator.getTelemetry());
```

Comment:
what are we asserting here then?

Reply:
Missed this. Metrics are handled by the sink builders. So not needed here. Removing this test.